gorillaz.go

```diff
@@ -139,12 +145,12 @@ func WithGrpcServerOptions(o ...grpc.ServerOption) Option {
 // It takes root at the current folder for properties file and a map of properties
 func New(options ...GazOption) *Gaz {
 	if initialized {
-		panic("gorillaz is already initialized")
+		Log.Warn("gorillaz is already initialized")
```

Comment:
I guess this should go away, it's a brand new Gaz, we don't need the initialized anymore

Reply:
yes